Metro/Metro_RP2350_Match3/match3_game/code.py

```diff
@@ -445,6 +490,11 @@ def atexit_callback():
                 # get the current mouse coordinates
                 coords = (mouse_tg.x, mouse_tg.y, 0)
 
+                # if the mouse point is within the exit
+                # button bounding box
+                if exit_btn.contains(coords):
+                    supervisor.reload()
```

Comment:
This was moved out of the `else` block for `if match3_game.cur_state != STATE_GAMEOVER:` just below here. 

Previously being in that `else` block meant that clicking within the bounds of the exit_btn would only be registered if it was in the GAMEOVER state. It looks to me like now it checks even when it's not GAMEOVER, but the exit button isn't visible so I think this could cause an invisible click area to run the `supervisor.reload()` if it were clicked.

Was there a reason for moving this out of the `else` block? And is there something else I haven't noticed that limits it to only take effect when the game is actually over?

Reply:
I moved the exit button to the edge of the display while the game was not in the GAMEOVER state so that the game could be exited using the mouse (the keyboard is disabled if all you have is the combo device attached). When the game is over the exit button is moved back to the location on top of the gameover displayio group.